docs/examples/basic_conceptual_guide/Part_1-model_deployment/Readme.md

```diff
@@ -0,0 +1,293 @@
+# Deploy models using Triton
+
+Any deep learning inference serving solution needs to tackle two fundamental challenges:
+* Managing multiple models.
+* Versioning, loading, and unloading models.
+
+## Managing multiple models
+
+The key challenge around managing multiple models is to build an infrastructure that can cater to the different requirements of different models. For instance, users may need to deploy a PyTorch model and TensorFlow model on the same server, and they have different loads for both the models, need to run them on different hardware devices, and need to independently manage the serving configurations (model queues, versions, caching, acceleration, and more). The Triton Inference Server caters to all of the above and more. 
+
+![multiple models](./img/1.PNG)
```

Comment:
Can we give these images more meaningful filenames across all of the examples? ex: `multiple_models.png` instead of `1.png`